packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-data-tile/OrganizationDataProtectionTile.component.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { DashboardTile, Links, LinkType } from '@ovhcloud/manager-components';
+import { OsdsChip } from '@ovhcloud/ods-components/react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { ODS_CHIP_SIZE } from '@ovhcloud/ods-components';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import {
+  DATA_PROTECTION_BACKUP_TITLE,
+  DATA_PROTECTION_RECOVERY_TITLE,
+} from '@/pages/dashboard/organization/OrganizationDashboard.constants';
+import { useManagedVcdOrganizationBackup } from '@/data/hooks/useManagedVcdOrganization';
+import Loading from '@/components/loading/Loading.component';
+import { veeamBackupAppName } from '@/routes/routes.constant';
+
+export default function DataProtectionTile() {
+  const { t } = useTranslation('dashboard');
+  const { id } = useParams();
+  const { shell } = React.useContext(ShellContext);
+  const { data: vcdBackup, isLoading } = useManagedVcdOrganizationBackup(id);
+  const [veeamHref, setVeeamHref] = React.useState('');
+
+  const backupStatus = vcdBackup?.data?.resourceStatus ?? 'error';
+
+  React.useEffect(() => {
+    shell.navigation
+      .getURL(veeamBackupAppName, '', {})
```

Comment:
```suggestion
      .getURL(veeamBackupAppName, `#/${id}-veeam-backup`, {})
```
To lend directly on the specific backup of the organization you are currently in

You should condition this to having a backup status

Reply:
Seen with the PO: always redirect on listing page